api/lib/infrastructure/repositories/certification-repository.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
           .whereRaw('"assessment-results"."createdAt" < "last-assessment-results"."createdAt"')
       );
 
-    if (latestAssessmentResultStatuses.includes('error') || latestAssessmentResultStatuses.includes('started')) {
+    if (latestAssessmentResultStatuses.includes('error') || latestAssessmentResultStatuses.includes(null)) {
```

Comment:
On ne garde pas le started?
```suggestion
    if (
     latestAssessmentResultStatuses.includes('error') 
     || latestAssessmentResultStatuses.includes('started')
     || latestAssessmentResultStatuses.includes(null)
    ) {
```

Reply:
'started' est un state de l'assessment mais pas un status de l'assessment-result

Reply:
tout à fait, c'est pour ça que je l'ai enlevé. L'équivalent en fait c'est de ne pas trouver d'assessment-result

Reply:
OK merci!